scripts/publish.sh

```diff
@@ -30,5 +45,12 @@ rm public/documentation/index.xml
 rm public/community/index.xml
 rm public/development/index.xml
 
+DESTINATION_STABLE_VERSION="public/documentation/stable"
+echo "Copying the stable documentation from ${STABLE_VERSION_FOLDER} to ${DESTINATION_STABLE_VERSION}"
+cp -r ${STABLE_VERSION_FOLDER} ${DESTINATION_STABLE_VERSION}
+
+echo "Add noindex meta tag to all HTML files in public/documentation/[0-9]*"
+find public/documentation/ -type f -regex "public/documentation/[0-9].*" -name "*.html" | xargs  sed -i '/<\/head>/i\<meta name="robots" content="noindex" \/>'
```

Comment:
I had to remove the slash at the end of the path:

    find public/documentation -type f ...

Also something is wrong with the sed invocation on OS X it seems. I'm getting

    sed: 1: "public/documentation/1. ...": extra characters at the end of p command

@agudian, any ideas on the latter?